app/models/cas_seeds/match_decision_reasons.rb

```diff
@@ -77,49 +80,56 @@ def run!
     end
 
     private def create_other_reason!
-      ::MatchDecisionReasons::Other.all.first_or_create! name: "Other"
+      reason = ::MatchDecisionReasons::Other.all.first_or_create! name: 'Other'
+      reason.update(referral_result: nil)
```

Comment:
With all of these, if these are always create (which I think they are), it will save a query to use the block notation to add the referral_results.